base/acme/src/main/java/org/dogtagpki/acme/database/LDAPDatabase.java

```diff
@@ -837,7 +834,7 @@ public ACMECertificate getCertificate(String certID) throws Exception {
         LDAPAttribute attr = entry.getAttribute(ATTR_CREATED);
         certificate.setCreationTime(dateFormat.parse(attr.getStringValues().nextElement()));
 
-        attr = entry.getAttribute(ATTR_USER_CERTIFICATE);
+        attr = entry.getAttribute(ATTR_USER_CERTIFICATE + ";binary");
```

Comment:
This change is fine to make in `LDAPDatabase`.  But maybe we should put the `";binary"` suffix in the definition of `ATTR_USER_CERTIFICATE`.  The current code adds the suffix for every use of the attribute (as it should) but putting it in `ATTR_USER_CERTIFICATE` should ensure that any new uses of the attribute includes the suffix (and eliminates a duplication).

Reply:
OK, we can do that.

Reply:
Done.